packages/one-app-runner/README.md

```diff
@@ -0,0 +1,258 @@
+# @americanexpress/one-app-runner
+
+> A command line interface (CLI) tool for running [One App](https://github.com/americanexpress/one-app) locally.
+
+**one-app-runner** allows you to work locally on your [Holocron Module](https://github.com/americanexpress/one-app/blob/master/docs/api/API.md) without having to clone [One App](https://github.com/americanexpress/one-app).
+
+It works by pulling a Docker image for One App and mounting your module to it as a volume.
+
+**Note:** If you are making changes to One App at the same time you are making changes to your Module(s) then you will still need to [clone One App, serve your module to it, and start it yourself](https://github.com/americanexpress/one-app).
+
+## 🤹‍ Usage
+
+Note that [Docker](https://www.docker.com/community-edition#/download) is required and must be installed before **one-app-runner** can be used.
+
+### Quick start
+
+In your Holocron Module directory:
+
+1. Install:
+```bash
+npm install --save-dev @americanexpress/one-app-runner
+```
+
+2. Add an `npm` start script and a runner config to either your module's `package.json`.
+
+##### package.json
+```json
+"scripts": {
+  "start": "one-app-runner"
+},
+"one-amex": {
+  "runner": {
+    "modules": ["."],
+    "rootModuleName": "frank-lloyd-root",
+    "moduleMapUrl": "https://example.com/cdn/module-map.json",
+    "dockerImage": "one-app-dev:5.0.0",
+    "parrotMiddleware": "./dev.middleware.js"
+  }
+}
+```
+
+In the runner config you will want to tell `one-app-runner` what module map and One App version to use, what your root module is, where your module is (current directory in this case), and optionally where your [Parrot](https://github.com/americanexpress/one-app/blob/master/docs/recipes/Mocking-API-Calls.md) middleware file is located.
+
+##  Command options reference
+
+**one-app-runner** can be configured via command options or a configuration object in `package.json`:
```

Comment:
should these be subheadings ?

Reply:
I'd add [required] or [optional] before the description of each command, ie at the beginning of the line, with the required ones first.

Reply:
Fixed in last commit.